app/src/main/java/org/inspire/breath/data/TextRecording.java

```diff
@@ -0,0 +1,11 @@
+package org.inspire.breath.data;
+
+import android.os.Parcel;
+
+// Mainly for debugging
+public class TextRecording extends Recording {
+    public TextRecording(int patientId) {
+        this.setKind("Text");
```

Comment:
Might make sense to make a public enum for the types of recording i.e. public enum RecordingKind {TEXT, HEARTRATE, BREATH} or even just {TEXT, AUDIO}